### PR TITLE
Implement batch operate util

### PR DESCRIPTION
**Pull request by pciporin25**
Adds `batch_operate` to the Airtop client

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -47,6 +47,7 @@ pydantic = ">= 1.9.2"
 pydantic-core = "^2.18.2"
 requests = "2.32.3"
 typing_extensions = ">= 4.0.0"
+pyee = "12.1.1"
```
Comment (pciporin25) [inline pyproject.toml]: Need to wait for `generators.yml` change to land for this to be updated

File: src/airtop/client.py [hunk 2]
```diff
@@ -33,6 +37,9 @@ class Airtop(BaseClient):
     httpx_client : typing.Optional[httpx.Client]
         The httpx client to use for making requests, a preconfigured client is used by default, however this is useful should you want to pass in any custom httpx configuration.
 
+    debug : typing.Optional[bool]
+        Whether to enable debug logging.
```
Comment (pciporin25) [inline src/airtop/client.py:37]: Added debugging ability to match Node SDK functionality

File: pyproject.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
```
Comment (pciporin25) [inline pyproject.toml:1]: Changes in this file and `poetry.lock` were copied from the [base branch](https://github.com/airtop-ai/airtop-python-sdk/pull/15) to resolve dependency-related compilation errors and verify the rest of the compilation succeeded

File: src/airtop/client.py [hunk 4]
```diff
@@ -104,7 +121,22 @@ def __init__(
...
+        if self.debug:
+            print(message)
+
+    def warn(self, message: str):
+        logging.warning(message)
+    
+    def error(self, message: str):
+        logging.error(message)
+
+    async def batch_operate(self, urls: List[BatchOperationUrl], operation: Callable[[BatchOperationInput], Awaitable[BatchOperationResponse]], config: Optional[BatchOperateConfig] = None) -> List[Any]:
```
Comment (pciporin25) [inline src/airtop/client.py:141]: I am only adding this ability to the `AsyncAirtop` client.  I don't know that the concurrency management involved is even possible to run in a synchronous environment.  All attempts I made at this failed.

Comment (pciporin25) [reply src/airtop/client.py:141]: I might pair with @kvindascr to see if we can get the synchronous version working.  But not sure that should block this going out anyway (we can release that as a follow-up)

**Review by kyleparrott — APPROVED** [review]

**Pull request merged**